src/signature/mod.rs

```diff
@@ -24,6 +29,51 @@ impl TryFrom<u8> for SignatureVersion {
     }
 }
 
+/// copied from wasm/wasm-crypto
+fn create_from_pair(pair: &[u8]) -> EdKeypair {
+    match EdKeypair::from_bytes(pair) {
+        Ok(pair) => return pair,
+        Err(_) => panic!("Provided pair is invalid."),
```

Comment:
I am against panicking unless it is for test code. I would suggest we use EdKeypair::from_bytes(pair) itself where we need to create a key from its pair and in that context handle the error.